ballista/scheduler/src/scheduler_server/grpc.rs

```diff
@@ -297,13 +298,24 @@ impl<T: 'static + AsLogicalPlan, U: 'static + AsExecutionPlan> SchedulerGrpc
         // TODO shouldn't this take a ListingOption object as input?
 
         let GetFileMetadataParams { path, file_type } = request.into_inner();
-        let file_format: Arc<dyn FileFormat> = match file_type.as_str() {
+        let file_format: Result<Arc<dyn FileFormat>, Status> = match file_type.as_str() {
             "parquet" => Ok(Arc::new(ParquetFormat::default())),
```

Comment:
I've noticed that in the [benchmark](https://github.com/apache/datafusion-ballista/blob/04766d5fd7cbb15d583f736979c11de06c720928/benchmarks/src/bin/tpch.rs#L830) there's slightly different `default` for parquet:
```rust
ParquetFormat::default().with_enable_pruning(Some(true))
```
I wonder if we need consistency or it's ok to leave it as is?

Reply:
I think pruning is default already, so in practice it will be the same.